docs/source/models/gbd2017/causes/measles/index.rst

```diff
@@ -123,9 +128,132 @@ outbreaks due to lack of vaccination in small communities.
 Data Description
 ----------------
 
-.. todo::
+.. list-table:: Definitions
+   :widths: 15 20 30
+   :header-rows: 1
+
+   * - State
+     - State Name
+     - Definition
+   * - S
+     - Susceptible
+     - Susceptible to measles
+   * - I
+     - Infected
+     - Infected with measles
+   * - R
+     - Recovered
+     - Recovered from measles
+
+
+.. list-table:: States Data
+   :widths: 20 25 30 30
+   :header-rows: 1
+   
+   * - State
+     - Measure
+     - Value
+     - Notes
+   * - S
+     - prevalence
+     - 1-prevalence_c341
+     - 
+   * - S
+     - excess mortality |br| rate
+     - 0
+     - 
+   * - S
+     - disabilty weights
+     - 0
+     -
+   * - I
+     - prevalence
+     - prevalence_c341
+     - 
+   * - I
+     - excess mortality |br| rate
+     - :math:`\frac{\text{deaths_c341}}{\text{population} \times \text{prevalence_c341}}`
+     - 
+   * - I
+     - disability weights
+     - disability_weight_s117 :math:`\times` |br| prevalence_s117+ |br| disability_weight_s118 :math:`\times` |br| prevalence_s118
+     - GBD assumes 50% of measles |br| cases as severe and other 50% |br| as moderate |br| [GBD-2017-YLD-Capstone-Appendix-1]_.
+   * - R
+     - prevalence
+     - 0
+     - Clearly room for improvement. |br| This is done to simplify the model |br| as the focus is on LSFF but not |br| on measles.
```

Comment:
Might be good to explicitly say that we don't have data for the number of recovered people, but the incidence rate we compute includes recovered as well as susceptible.